hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -276,9 +277,9 @@ private void flushTransactions() {
             AtomicReference<String> lastTraceId = new AtomicReference<>();
             readyBuffer.iterator().forEachRemaining((entry) -> {
               try {
-                OMResponse omResponse = entry.getResponse().getOMResponse();
```

Comment:
nit: I don't think `omResponse` is used in any lambda. You probably just need to move this line to outside the try block (move up 2 lines). Another `AtomicReference ` is not needed.

Reply:
I'm not sure if I'm understanding correctly, this is the inital error IDE gives me.
<img width="978" alt="Screenshot 2023-01-05 at 10 33 18 AM" src="https://user-images.githubusercontent.com/14933944/210688523-e68793af-fa7a-4d70-9877-3be0185e41e4.png">

Reply:
I was wrong actually, thought it `omResponse` is used only inside the for loop. It's alright then.